packages/react/src/Tooltip/Tooltip.features.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import type {Meta} from '@storybook/react'
+import {Button, Box, Link, Stack} from '..'
+import {Tooltip} from './Tooltip'
+import {BookIcon} from '@primer/octicons-react'
+
+export default {
+  title: 'Components/Tooltip/Features',
+  component: Tooltip,
+} as Meta<typeof Tooltip>
+
+export const AnchorHasMargin = () => (
+  <Box padding="spacious">
+    <Tooltip text="Tooltip is still centered">
+      <Button sx={{marginLeft: 3}}>Button has 16px margin Left</Button>
+    </Tooltip>
+  </Box>
+)
+
+export const LabelType = () => (
+  <Box padding="spacious">
+    <Tooltip text="Contribution Documentation for 'Primer React'" type="label">
+      <Link href="https://github.com/primer/react/contributor-docs/CONTRIBUTING.md">
+        <BookIcon />
+      </Link>
+    </Tooltip>
+  </Box>
+)
```

Comment:
FYI, this story doesn't seem to be working as expected (compared to the PRC version).


https://github.com/primer/brand/assets/13340707/06e9e5aa-f4ec-4834-94fb-940becbed832

Reply:
I'm also not sure we'd want to encourage using tooltip this way., using an Octicon inside a Link component. cc. @nsolerieu to keep me honest though.

Reply:
What @rezrah recorded definitely looks odd

- Having octicon to a link is okay **WITH** a label
- Having **ONLY** an octicon in a link is not okay
- Using a tooltip on a link is okay as long as the link is setup properly (label + icon + arrow OR label + arrow)

Reply:
@nsolerieu @rezrah 

Aren't we planning to use tooltips on the social media icon links in the footer? Wouldn't that be an example of a visible label being replaced by a Tooltip?

Reply:
I agree this example looks strange with with the arrow that automatically gets added by `Link`. How do you recommend I move forward?
- Update this story to use the correct component for rendering an icon-only link or button
- Remove this story and the ability to use a Tooltip as a label
- Something else

Reply:
I'd recommend using a Button instead @mperrotti... 

Example:

<img width="724" alt="Screenshot 2024-04-16 at 09 35 27" src="https://github.com/primer/brand/assets/13340707/91966622-12af-45a4-a78a-a36c4f1c9ce3">